java/vector/src/main/codegen/templates/ComplexCopier.java

```diff
@@ -47,6 +47,7 @@ private static void writeValue(FieldReader reader, FieldWriter writer) {
       switch (mt) {
 
       case LIST:
+      case TUPLE_2:
```

Comment:
Not that I could figure out... I think this goes back to the problem with the java minor types - I believe @julienledem was planning to remove them eventually.